src/index-metadbs/README.md

```diff
@@ -0,0 +1,41 @@
+# Static Index Meta-Databases
+
+## Goal
+
+This folder hosts static versions of Index Meta-Databases for those providers that only have one main sub-database (or very few sub-databases) and do not wish to maintain an Index Meta-Database themselves (Index Meta-Databases are described in the [OPTIMADE specifications](https://github.com/Materials-Consortia/OPTIMADE/blob/develop/optimade.rst)).
+
+Note: while providing an (empty) Index Meta-Database is not required by the OPTIMADE API specifications, it is instead required in order to be listed in this List of Providers ([providers.optimade.org](http://providers.optimade.org)).
+
+## When should I use a static index meta-database hosted in this repository
+
+If you add a new provider and do not wish to host an Index Meta-Database yourself, feel free to add an Index Meta-Database here.
+
+However, if your Index Meta-Database is rapidly changing (e.g. because the list of sub-databases changes often), please host it yourself to avoid multiple commits and pull requests to this repository at every change.
+Rare changes are instead of course welcome, e.g., if you have to change your DNS name or the description, or if you need just to add one static new sub-database, like a test database.
+
+Note that "changes" here refer solely to changes to the *list of sub-databases*; the content of each sub-database can change at any time without the need to modify the Index Meta-Database.
+
+## Instructions on how to add a new static Index Meta-Database
+1. Go in the folder `src/index-metadbs`, copy one of the existing folders (e.g. `cod`) to a new folder, where the folder name should be the identifier of your provider (we will use `exmpl` here and in the following).
```

Comment:
I think it is a bit inelegant to tie this document to an actual specific provider (cod). Can you instead create a 'template' directory that people can copy?

Reply:
Sure

Reply:
Done in f27908a9643b9259a53b9cec0b468fef60f70829